modules/extensions-support/src/main/java/org/mule/module/extension/internal/model/AbstractAnnotatedModelEnricher.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.model;
+
+import org.mule.extension.introspection.declaration.fluent.BaseDeclaration;
+import org.mule.extension.introspection.declaration.spi.ModelEnricher;
+import org.mule.module.extension.internal.introspection.AnnotationsBasedDescriber;
+import org.mule.module.extension.internal.model.property.ImplementingTypeModelProperty;
+
+import java.lang.annotation.Annotation;
+
+/**
+ * Base class for implementations of {@link ModelEnricher} which provides utility methods
+ * for enriching {@link BaseDeclaration declarations} constructed from an annotated model
+ *
+ * @see AnnotationsBasedDescriber
+ * @since 4.0
+ */
+public abstract class AbstractAnnotatedModelEnricher implements ModelEnricher
+{
+
+    /**
+     * Extracts (if present) an {@link Annotation} from class backing the {@code declaration}.
```

Comment:
Remove the "(if present)" part as that is already documented in the returns section